certificate-manager/src/main/java/io/strimzi/certs/SecretCertProvider.java

```diff
@@ -26,6 +27,8 @@
 
     public static final String DEFAULT_KEY_KEY = "tls.key";
     public static final String DEFAULT_KEY_CERT = "tls.crt";
+    public static final String DEFAULT_KEY_STORE = "tls.str";
```

Comment:
Is this the key under which it is stored in the secrets? I think we should be more verbose / clear ... e.g.: `tls.p12` and `truststore` or `truststre.jks` and maybe just `password` or `password.txt` or `truststore.password`?

Reply:
I changed the definition we have in the `Ca` class into `truststore.p12` and `truststore.password` as you suggested; they makes more sense to me. I changed the ones here just as `tls.p12` and `tls.password` but not using "truststore" because this class is more generic even for putting "keystore" into a Secret.

Reply:
Again, I think we need to properly distinguish that otherwise we will get crazy about it one week after we merge the PRs. It is a big difference and it needs to be clear what you are working with. The X509 certificates are also not named just `tls1` and `tls2` but make it clear what is what.

Reply:
The method that uses these consts is just `SecretCertProvider.createSecret` (one of the overloads) which is just used in a couple of tests in the `SecretCertProviderTest` class. I don't think that names are so important for this reason. So I think that the best option is just removing them and using the other `createSecret` overload in the test class which passes values for keys. Because this defaults are never used through the operators and because they key has to be meaningful, I think it makes sense to remove them.

Reply:
Here is the thing which confuses me - the names of the fields in the secret are a form of API. If I write my application to use them it should not change. So why to run the actual field name with one name and the tests with different name?

Reply:
So with the latest commit, I removed them and used the same keys as the actual operator uses but in the tests.